packages/gateway/src/features/registration/schema.graphql

```diff
@@ -274,6 +274,7 @@ input BirthRegistrationInput {
   lastPreviousLiveBirth: Date # Not in BDRIS2
   createdAt: Date
   updatedAt: Date
+  ageOfIndividualInYears: String
```

Comment:
Which individual does this field reference?

Reply:
I'm still working on it.